python/semantic_kernel/connectors/ai/chat_request_settings.py

```diff
@@ -19,6 +19,7 @@ class ChatRequestSettings:
     max_tokens: int = 256
     token_selection_biases: Dict[int, int] = field(default_factory=dict)
     stop_sequences: List[str] = field(default_factory=list)
+    chat_system_prompt: str = "Assistant is a large language model."
```

Comment:
hugest of nits: In dotnet - this got defined on Complete Request Settings but not Chat Request Settings. I think that makes sense because when using ChatRequestSettings, you are typically supplying the messages List which will contain the system prompts. Text completion is the one that needs the override ability.

Reply:
removed from chat_request_settings